src/main/webapp/resources/js/pages/analysis/preview.utils.js

```diff
@@ -43,12 +46,12 @@ export const panelHeading = (
     fileSizeBytes
   }
 ) =>
-  `<div class="panel-heading">
-     <h5 class="panel-title">
+  `<div style="margin-bottom: 20px">
+     <h5>
```

Comment:
`h5` should be done using the ant typography heading.

Reply:
This file is going to be deleted. It is no longer used

Reply:
Good.  I was about to leave you a long comment on it about how I hated everything in it.  **seriously**

Reply:
Removed in 17c23fd74ab4bdcfe6c55f80cbe3c82226c59dab